utils.js

```diff
@@ -194,6 +194,23 @@ CTLEventUtils.readURLParams = function(queryString) {
 };
 
 /**
+ * @param eventsList = A list of events to select from
+ *
+ * @param eventID = the guid from Bedeworks that identifies the event
+ *
+ * @return = An array containing the single event that matches the ID
+ */
+CTLEventUtils.getEventByID = function(eventsList, eventID) {
+    if (eventsList == null || eventID == null) {return [];}
```

Comment:
I would change this to:
```
if (!eventsList || !eventID) {
    return [];
}
```

Using `==` instead of `===` in JavaScript can lead to confusion so it's best to avoid it. If your intention is to return an empty array if eventsList or eventID are either null or empty, use the above code. Otherwise switch to triple equals.